lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -509,6 +509,7 @@ def __init__(self):
             allocated=dict(type='bool', default=True),
             restarted=dict(type='bool', default=False),
             started=dict(type='bool', default=True),
+            volume_type=dict(choices=['ssd', 'hdd'], type='str', required=True, default='hdd'),
```

Comment:
`required=True` can be removed since there is default value.

Reply:
actually i was wondering if "volume_type" is the best name of this parameter.
Perhaps it should be "os_disk_type"? In theory we could also add data disks to the vm.

Reply:
Seems like Standard_LRS is Standard HDD whereas Premium_LRS is Premium SSD 
So having SSD vs HDD distinct from Standard and Premium would make sense
Also should consider the new type Ultra (just SSD)

Reply:
thanks for your contribution. agree on name of "os_disk_type", and it's better to use standard_lrs and premium_lrs here, in case future there might be ssd for standard price tier. and  pls also update documentation in this module.